### PR TITLE
Add Max 6675 k type thermocouple temperature sensor

### PR DESCRIPTION
**Pull request by ermaya**

### CODE REVIEW

File: libraries/SuplaDevice/supla/sensor/MAX6675_K.h [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	  
+  }
+
+
+  void onInit() {
+    pinMode(_pin_CS, OUTPUT);
+    pinMode(_pin_CLK, OUTPUT); 
+    pinMode(_pin_DO, INPUT);
+
+    digitalWrite(_pin_CS, HIGH);
```
Comment (klew) [inline libraries/SuplaDevice/supla/sensor/MAX6675_K.h:70]: Please move digitalWrite before pinMode

File: libraries/SuplaDevice/supla/sensor/MAX6675_K.h [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    pinMode(_pin_CS, OUTPUT);
+    pinMode(_pin_CLK, OUTPUT); 
+    pinMode(_pin_DO, INPUT);
+
+    digitalWrite(_pin_CS, HIGH);  
+  
+    channel.setNewValue(getValue());
+  }
+
+  byte spi_read(void) {
```
Comment (klew) [inline libraries/SuplaDevice/supla/sensor/MAX6675_K.h:75]: Please use camel case for method names: spiRead

File: libraries/SuplaDevice/supla/sensor/MAX6675_K.h [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+ public:
+  MAX6675_K(uint8_t pin_CLK,uint8_t pin_CS,uint8_t pin_DO) {
+    _pin_CLK = pin_CLK;
+    _pin_CS = pin_CS;
+	  _pin_DO = pin_DO;
+  }
+
+  double getValue() {
+	  
+	    double value;
```
Comment (klew) [inline libraries/SuplaDevice/supla/sensor/MAX6675_K.h:44]: Fix indentation in all files

File: libraries/SuplaDevice/supla/sensor/MAX6675_K.h [hunk 4]
```diff
@@ -0,0 +1,103 @@
+/*
```
Comment (klew) [inline libraries/SuplaDevice/supla/sensor/MAX6675_K.h:1]: Please split this file for .h and .cpp

File: libraries/SuplaDevice/examples/Supla_MAX6675_No_SSL/Supla_MAX6675_No_SSL.ino [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+   * Otherwise you will get "Channel conflict!" error.
+   */
+
+  new Supla::Sensor::MAX6675_K(15, 13, 12);  //  MAX6675(pin_CLK, pin_CS, pin_DO)
+
+  /* By default this library uses secured connection with SUPLA server. You can switch it of by writing below line 
+   * Works only with ESP8266 boards!
+   */
+
+  wifi.enableSSL(false);
```
Comment (klew) [inline libraries/SuplaDevice/examples/Supla_MAX6675_No_SSL/Supla_MAX6675_No_SSL.ino:50]: remove it, and rename example to something like: "Thermometer_MAX6675" (don't use "Supla" name in examples - they are anyway showed by Arduino under SuplaDevice "folder".

File: libraries/SuplaDevice/examples/Supla_MAX6675_No_SSL/Supla_MAX6675_No_SSL.ino [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+*/
+
+#include <SPI.h>
+#include <SuplaDevice.h>
+
+
+// Add include to MAX6675_K sensor
+#include <supla/sensor/MAX6675_K.h>
+
```
Comment (klew) [inline libraries/SuplaDevice/examples/Supla_MAX6675_No_SSL/Supla_MAX6675_No_SSL.ino:23]: please add netowrk options for Arudino Mega ehternet shield and make sure it compiles on AVR board target.

**Review by klew — CHANGES_REQUESTED** [review]

File: libraries/SuplaDevice/supla/sensor/MAX6675_K.h (hunk 1 above)
Comment (ermaya) [reply libraries/SuplaDevice/supla/sensor/MAX6675_K.h:70]: thanks for the suggestions, it only took you 6 months to raise.
for me the theme is obsolete.

Comment (klew) [reply libraries/SuplaDevice/supla/sensor/MAX6675_K.h:70]: Please forgive my tardiness.
I think this sensor will be beneficial. So please confirm if you'd like to add requested correcions, or withdraw this pr.

Comment (ermaya) [reply libraries/SuplaDevice/supla/sensor/MAX6675_K.h:70]: @ krycha88 will do a new pull request

**Pull request closed without merging**